routes/auth.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const router = express.Router();
+const knex = require('../database/knex');
+const bcrypt = require('bcryptjs');
+
+router.route('/dashboard')
+  .get(function (req, res) {
+    res.render('auth/dashboard')
+  });
+
+router.route('/login')
+  .get(function (req, res) {
+    res.render('auth/login')
+  });
+
+router.route('/register')
+  .get(function (req, res) {
+    res.render('auth/register')
+  });
+
+router.route('/logout')
+  .post(function (req, res) {
+
```

Comment:
This is missing the call to `req.logout()` to actually logout the user (clear session and remove cookie).